tox.ini

```diff
@@ -73,21 +79,45 @@ commands =
 [testenv:min-deps]
 # Run Python unit tests with the minimal dependencies installed
 deps =
-    {[testenv]rascaline-dependencies}
+    {[testenv]equistore-core-requirement}
     pytest
 
 commands =
     pytest --import-mode=append {posargs}
 
 
+[testenv:torch-tests]
+deps =
+    {[testenv]equistore-torch-requirement}
+
+    pytest
+    numpy
+    torch
+    ase
+
+    cmake
+    wheel
+
+changedir = python/rascaline-torch
+commands =
+    # install rascaline-torch
+    pip install . {[testenv]build-single-wheel} --force-reinstall
+    # run the unit tests
+    pytest --import-mode=append --assert=plain {posargs}
```

Comment:
@PicoCentauri check this if this can be removed.

Reply:
I played around but didn't found anything. Lets keep this and maybe pytets will solve this in a future version.